yarn/datadog_checks/yarn/data/conf.yaml.example

```diff
@@ -41,6 +41,10 @@ instances:
     # Allowed yarn keys: applicationType, applicationTags, name, queue, user
     # By default, the application name is collected with the prefix app_name.
 
+    # Whether to enable SSL certificate verification for HTTP requests. Defaults to true, you may
+    # need to set to false when using self-signed certs
+    # ssl_verify: true
```

Comment:
When it's a true or false value. it's usually better to have param starting with `is_*`. For example `is_ssl_verification_enable`

Reply:
You're right, but a lot of integrations already use `ssl_verify`. I like the idea of using `is_*`. We should discuss this issue for the soon-to-be-implemented compatibility layer.

>Be the change you wish to see!

Reply:
I agree with keeping `ssl_verify`

Reply:
Ah ok. I didn't know it was elsewhere. Fair point regarding grouping.